strings/charoccurrence.go

```diff
@@ -0,0 +1,22 @@
+// charoccurrence.go
+// description: Chars occurrence in the sentence
+// details: An algorithm which counts the number
+// of times a character occurred in a string.
```

Comment:
```suggestion
// description: An algorithm which counts the number
// of times a character occurred in a string.
```

Reply:
Now the last thing is ok. :)

Reply:
I think you don't know much about how the diffs work 🙂 Essentially, the `-` sign on the line just says to remove those lines and `+` means to add those lines. Since in this diff, we have 2 `+` and 3 `-` it means you needed to remove a line but you just added two lines.

To help, there is actually a commit suggestion button on suggested text which you can use to commit directly.

Reply:
I know but my place is not good for coding now. sorry :)